src/transformers/models/vit/modeling_tf_vit.py

```diff
@@ -677,6 +676,8 @@ def call(
         >>> inputs = feature_extractor(images=image, return_tensors="tf")
         >>> outputs = model(**inputs)
         >>> last_hidden_states = outputs.last_hidden_state
+        >>> list(last_hidden_states.shape)
+        [1, 197, 768]
         ```"""
```

Comment:
Also wasn't sure about the style here, keep ``` and """ in the same line or split to two lines

Reply:
It should be more or less a style choice. In the model files, we usually keep it as **```"""**